sourcemash/frontend/templates/categorizer.tpl

```diff
@@ -0,0 +1,83 @@
+<div id="splash">
```

Comment:
I think the red bars a bit excessive. They're not headers for anything like how we use them on Browse.